### PR TITLE
fix(FEC-11128): X button on floating player not working

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

when an ad is playing, the dismiss button is showing but not working.
made all left-controls and right-controls children clickthrough and excepted ad-notice.

Solves FEC-11128

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Review by RoyBregman — APPROVED** [review]

File: src/components/top-bar/_top-bar.scss [hunk 1]
```diff
@@ -28,6 +30,10 @@
     &:empty {
       margin: 0;
     }
+
+    .ad-notice {
```
Comment (yairans) [inline src/components/top-bar/_top-bar.scss:34]: ad notice has its own css file. do it there

File: src/components/top-bar/_top-bar.scss [hunk 2]
```diff
@@ -45,6 +51,11 @@
       }
     }
   }
+
+  .left-controls *,
+  .right-controls * {
+    pointer-events: auto;
```
Comment (yairans) [inline src/components/top-bar/_top-bar.scss:57]: the top-bar itself it also a container so add this in it

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/components/top-bar/_top-bar.scss (hunk 2 above)
Comment (yairans) [reply src/components/top-bar/_top-bar.scss:57]: with scss you can do like:
```
  &.disabled {
    pointer-events: none;
    * {
       pointer-events: auto;
    }
  }```

File: src/components/top-bar/_top-bar.scss (hunk 1 above)
Comment (lianbenjamin) [reply src/components/top-bar/_top-bar.scss:34]: @yairans it's not working when adding this to ad-notice.scss. the adding of pointer-events:auto to top-bar's children is stronger. we needed to add another level to make this one stronger.

File: src/components/top-bar/_top-bar.scss (hunk 2 above)
Comment (RoyBregman) [reply src/components/top-bar/_top-bar.scss:57]: We tried first what you suggested but it doesn't fix the issue correctly. The problem is that the left and right controller also have dimensions due to its children, so it is also grabbing the clicks. That is why we lowered it another level

**Review by yairans — APPROVED** [review]

**Pull request merged**